index.rb

```diff
@@ -66,6 +66,25 @@ def show(**kwargs)
   end
 end
 
+def seetickets_parser(event)
+  title = event.css('p.title').text
+
+  return nil if title =~ /private event/i
```

Comment:
Returning nil here doesn't feel great now that it's an actual function :thinking:

Reply:
You could pass in the nokogiri collection:

```ruby
seetickets_events(Nokogiri(html).css('div.seetickets-list-event-container'))
```

I think I would keep the title check in the venue-specific code:

```ruby
seetickets_events(...).reject do |event|
  event.title.match?(/private event/i)
end
```